test/test_main.py

```diff
@@ -6,30 +6,30 @@
 from interfaces.TaggingOperation import TaggingOperation
 
 
-def getMessage(transformation, impl_name):
-    return "For {transformation}, no transformation or test case found for {impl_name}".format(
-        transformation=transformation, impl_name=impl_name
+def getMessage(transformation_type, impl_name):
```

Comment:
We can remove "impl_name" if we are not using it.

Reply:
Done..